### PR TITLE
Small Edit to hcl.md

### PR DESCRIPTION
**Pull request by stubbybubby**
* Expanded Introduction
* Finished unfinished sentences
* Moved some sentences

### CODE REVIEW

**Comment by tasket** [comment]
Thanks!

File: hardware/hcl.md [hunk 1]
```diff
@@ -13,33 +13,34 @@ redirect_from:
...
 -   the best achievable Qubes security level (green columns in HVM, IOMMU, TPM)
 -   and general machine compatibility (green columns in Qubes version, dom0 kernel, remarks).
 
 Generating and Submitting New Reports
 -------------------------------------
 
-In order to generate a HCL report in Qubes, simply open a terminal in dom0 (KDE: start-menu -\> System Tools -\> Konsole or Terminal Emulator)
- and run `qubes-hcl-report <vm-name>`, where `<vm-name>` is the name of the VM to which the generated HCL files will be saved.
- (Note: If you are working with a new Qubes installation, you may need to update your system in order to download this script.)
+In order to generate a an HCL report in Qubes, simply open a terminal in dom0 (KDE: start-menu -\> System Tools -\> Konsole or Terminal Emulator)
```
Comment (andrewdavidwong) [inline hardware/hcl.md]: Typo?

Comment (stubbybubby) [inline hardware/hcl.md]: The acronym is pronounced "Aych See Ell". The letter in the acronym is H but it's the sound that counts.

Comment (andrewdavidwong) [inline hardware/hcl.md]: That's not what I'm referring to. I'm referring to "a an."

Comment (stubbybubby) [inline hardware/hcl.md]: ...

;_;

**Pull request merged**